data/json_common.h

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2024 GARDENA GmbH
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ *
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v20.html
+ * The Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.php.
```

Comment:
How about starting to use SPDX identifiers? If you agree, I'll create a PR to migrate all existing code.